js/bundle.go

```diff
@@ -39,6 +40,23 @@ type Bundle struct {
 	moduleResolver  *moduleResolver
 }
 
+// TODO: this is to be removed once this is not a warning and it can be moved to the registry
+func (b *Bundle) checkMetricNamesForPrometheusCompatibility() {
+	const (
+		nameRegexString = "^[a-zA-Z_][a-zA-Z0-9_]{1,128}$"
```

Comment:
I didn't see a length limit on Prometheus' metric names in their documentation, but [OTel limits it to 63 chars](https://opentelemetry.io/docs/specs/otel/metrics/api/#instrument-name-syntax).

I'm not sure if we should follow whatever Prometheus' limit is, or OTel's, but 63 chars is way too restricting. So... neither?

Reply:
True, 63 chars ought to be enough for anybody.:tm: :smile:

It's probably a safe limit to impose. Even the browser's long Web Vital metrics like `webvital_first_contentful_paint_good` is 36 chars, and going much above that would be messy to show in our CLI and Cloud UIs.

So if we want to be strict with OTel compatibility, we might also limit it to 63.